app/models/container_node.rb

```diff
@@ -78,4 +79,19 @@ def ipaddress
   def cockpit_url
     URI::HTTP.build(:host => ipaddress, :port => 9090)
   end
+
+  supports :common_logging do
+    unless ext_management_system.respond_to?(:common_logging_route_name)
+      unsupported_reason_add(:common_logging, _('This provider type does not support common_logging'))
+    end
+  end
+
+  def common_logging_query
+    nil # {}.to_query # TODO
+  end
+
+  def common_logging_path
+    hostname = name.split('.')[0]
```

Comment:
First, logs almost never have perfect metadata, and so while we strive to have an FQDN in the hostname field, as is the case already in OpenShift 3.4, some parts of the system send an FQDN in, some just $(hostname -s).  So, yes, collisions are there, but usually there is other metadata in the logs that help to distinguish the log entries.